esp32/esp32.ino

```diff
@@ -1,18 +1,48 @@
 #include "sensorsManager.h"
+#include "networkManager.h"
 
 SensorsManager sensorsManager;
+WiFiController wc = WiFiController("Mot de passe = free", "azera123");
```

Comment:
perhaps not sharing your phone hotspot password on github might be a good idea :)

Reply:
WiFiController wc = WiFiController("your_SSID", "your_wifi_PASSWORD");